plugin.php

```diff
@@ -3,22 +3,16 @@
 Plugin Name: Conditional URL Advertisements
 Plugin URI: https://github.com/HeroCC/yourls-conditional-urlads
 Description: Conditionally send shortlinks through various link monetizing services
-Version: 1.1
+Version: 1.2.1
 Author: HeroCC
 Author URI: https://herocc.com/
 */
 
 if( !defined( 'YOURLS_ABSPATH' ) ) die();
 
-define( 'TRIGGERS', array('a/', 'f/', 'o/') ); // Add any possible trigger to use here
+define( 'TRIGGERS', array('a/', 'f/', 'o/', 'l/') ); // Add any possible trigger to use here
 
-define( 'ADFLY_ID', '2777408' ); // Replace this with your Adfly ID
-define( 'ADFOCUS_ID', '287608' ); // Replace this with your Adfoc.us ID
-define( 'OUO_ID', '0IqYvHOo' ); // You get the drill
-
-define( 'ADFLY_DOMAIN', 'https://adf.ly' ); // If you have a custom Adfly domain, replace this with it
-define( 'ADFOCUS_DOMAIN', 'https://adfoc.us' ); // Same for this
-define( 'OUO_DOMAIN', 'https://ouo.io' ); 
+include 'user-config.php';
```

Comment:
This is a good change too. If you'd like to make it another PR, that would be nice so it comes in as another commit rather than the single merge here.

Reply:
你的意思是这个更新附着在另外一个推送吗？

Reply:
Correct, feel free to move this part to another PR.